CBLClient/Apps/JavaRequestHandlers/src/main/java/com/couchbase/mobiletestkit/javacommon/RequestHandler/DatabaseRequestHandler.java

```diff
@@ -45,6 +45,11 @@ public Database create(Args args) throws CouchbaseLiteException {
         if (config == null) {
             config = new DatabaseConfiguration();
         }
+        String dbDir = config.getDirectory();
+        if(dbDir == null || dbDir.equals("/")){
```

Comment:
I think it is worth to comment here why dbDir.equals("/") is not allowed.

Reply:
added.